lightning/src/util/events.rs

```diff
@@ -343,6 +343,9 @@ pub enum Event {
 	/// This event is generated when a payment has been successfully forwarded through us and a
 	/// forwarding fee earned.
 	PaymentForwarded {
+		/// The channel between the source node and us. Optional because PaymentForwarded event can 
+		/// be emitted even after the channel has been closed.
```

Comment:
This should mention something about old versions of LDK.

Reply:
I'll just say it's optional because old versions of LDK do not serialize this field. And I think we should mention a version number to make it more concrete. If yes, is it `v0.0.106`?

Reply:
Yep, best to say "versions prior to 0.0.107" or "versions up to 0.0.106" or whatever.